src/main/java/uk/gov/digital/ho/hocs/workflow/client/caseworkclient/CaseworkClient.java

```diff
@@ -113,17 +113,15 @@ public UUID createCaseNote(UUID caseUUID, String type, String text) {
         return response;
     }
 
-    public UUID createStage(UUID caseUUID, String stageType, UUID teamUUID, UUID userUUID, String allocationType) {
-        CreateCaseworkStageRequest request = new CreateCaseworkStageRequest(stageType, teamUUID, userUUID, allocationType);
+    public UUID createStage(UUID caseUUID, CreateCaseworkStageRequest request) {
         CreateCaseworkStageResponse response = restHelper.post(serviceBaseURL, String.format("/case/%s/stage", caseUUID), request, CreateCaseworkStageResponse.class);
         log.info("Created Stage: {} for Case {}", response.getUuid(), caseUUID);
         return response.getUuid();
     }
 
-    public void recreateStage(UUID caseUUID, UUID stageUUID, String stageType) {
-        RecreateCaseworkStageRequest request = new RecreateCaseworkStageRequest(stageUUID, stageType);
-        restHelper.put(serviceBaseURL, String.format("/case/%s/stage/%s/recreate", caseUUID, stageUUID), request, Void.class);
-        log.info("Recreated Stage: {} for Case {}", stageUUID, caseUUID);
+    public void recreateStage(UUID caseUUID, RecreateCaseworkStageRequest recreateStageRequest) {
+        restHelper.put(serviceBaseURL, String.format("/case/%s/stage/%s/recreate", caseUUID, recreateStageRequest.getStageUUID()), recreateStageRequest, Void.class);
+        log.info("Recreated Stage: {} for Case {}", recreateStageRequest, caseUUID);
```

Comment:
personally i'm squeamish enough at debug, but we shouldn't be logging whole DTOs at Info, it goes into Kibana and who knows where else. I appreciate this particular DTO is pretty harmless but there could be any sort of data in the future.